pages/feed/index.tsx

```diff
@@ -85,6 +85,29 @@ const Feeds: WithPageLayout<HighlightSSRProps> = (props: HighlightSSRProps) => {
     { name: "Highlights", count: highlights_count ?? 0 },
   ];
 
+  useEffect(() => {
+    const queryParams = new URLSearchParams(window.location.search);
```

Comment:
```suggestion
    const { new: newHighlight, signIn: signInRequired } = router.query;
```

Hey @diivi, I'd suggest we destructure the queries from `next/router` here like so..